src/sdk/modules/validators/ownableValidator/ownableValidator.test.ts

```diff
@@ -0,0 +1,325 @@
+import { getAddOwnableValidatorOwnerAction } from "@rhinestone/module-sdk"
+import {
+  http,
+  type Account,
+  type Address,
+  type Chain,
+  type Hex,
+  type PublicClient,
+  encodeAbiParameters,
+  encodeFunctionData,
+  encodePacked,
+  getAddress,
+  parseAbi,
+  zeroAddress
+} from "viem"
+import { afterAll, beforeAll, describe, expect, test } from "vitest"
+import { TEST_CONTRACTS } from "../../../../test/callDatas"
+import { toNetwork } from "../../../../test/testSetup"
+import {
+  fundAndDeployClients,
+  getTestAccount,
+  killNetwork,
+  toTestClient
+} from "../../../../test/testUtils"
+import type { MasterClient, NetworkConfig } from "../../../../test/testUtils"
+import addresses from "../../../__contracts/addresses"
+import {
+  type NexusClient,
+  createNexusClient
+} from "../../../clients/createNexusClient"
+import { parseModuleTypeId } from "../../../clients/decorators/erc7579/supportsModule"
+import {
+  type ToK1ValidatorModuleReturnType,
+  toK1ValidatorModule
+} from "../k1Validator/toK1ValidatorModule"
+import {
+  type ToOwnableValidatorModuleReturnType,
+  toOwnableValidatorModule
+} from "./toOwnableValidationModule"
+
+describe("modules.ownableValidator", async () => {
+  let network: NetworkConfig
+  let chain: Chain
+  let bundlerUrl: string
+
+  // Test utils
+  let testClient: MasterClient
+  let account: Account
+  let nexusClient: NexusClient
+  let nexusAccountAddress: Address
+  let recipient: Account
+  let recipientAddress: Address
+  let ownableValidatorModule: ToOwnableValidatorModuleReturnType
+  let k1ValidatorModule: ToK1ValidatorModuleReturnType
+  beforeAll(async () => {
+    network = await toNetwork("FILE_LOCALHOST")
+
+    chain = network.chain
+    bundlerUrl = network.bundlerUrl
+    account = getTestAccount(0)
+    recipient = getTestAccount(1)
+    recipientAddress = recipient.address
+
+    testClient = toTestClient(chain, getTestAccount(5))
+
+    nexusClient = await createNexusClient({
+      signer: account,
+      chain,
+      transport: http(),
+      bundlerTransport: http(bundlerUrl)
+    })
+
+    nexusAccountAddress = await nexusClient.account.getCounterFactualAddress()
+    await fundAndDeployClients(testClient, [nexusClient])
+
+    ownableValidatorModule = await toOwnableValidatorModule({
+      address: TEST_CONTRACTS.OwnableValidator.address,
+      client: nexusClient.account.client as PublicClient,
```

Comment:
I don't like that we make the user do this. Can we add a 'toInitData' / getInitData helper for each module? This way we keep the decorators mostly account agnostic

Reply:
Yea sure, will fix this in another PR, there are other improvements we can do as well.

Reply:
Awesome 😎